client/main.scss

```diff
@@ -38,7 +38,7 @@ h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
   min-height:calc(100vh - 70px - 65px);
 }
 
-html, body, #react-root {
+html, body, #react-root, [data-reactroot] {
```

Comment:
@richsilv Very minor, (to help my understanding really) do we need both `#react-root` & `[data-reactroot]` ?